File: pgm/ABC_Java/ABC_437_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        String[] arr = reader.readLine().split (" ");
        reader.close();
        final int one = 1;
        final int zero = 0;
        final int twelve = 12;
        int A = Integer.parseInt (arr[zero]);
        int B = Integer.parseInt (arr[one]);
        System.out.println (A * twelve + B);
        System.exit (0);
    }
}

File: pgm/ABC_C#/ABC_437_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split(' ');
        const int one = 1;
        const int twelve = 12;
        const int zero = 0;
        int A = int.Parse(arr[zero]);
        int B = int.Parse(arr[one]);
        Console.WriteLine(twelve * A + B);
        Environment.Exit(0);
    }
}